src/components/Vault/VaultDepositForm.tsx

```diff
@@ -90,6 +92,7 @@ const VaultDepositForm: FC<VaultProps> = (props) => {
   })
   // Preview deposit method
   const { isLoading: isLoadingPreview } = useContractRead({
+    chainId,
```

Comment:
We must pass chainId to all read functions if we want disconnected users to be able to switch chains. I'm not thrilled about this and have ideas for cleaning it up, but for now this is required.